client/src/components/Signup/Signup.js

```diff
@@ -102,64 +141,75 @@ export default function SignUp() {
+                            classes: {root: classes.textFieldSelectLabel} 
+                          }}
+                          value={formValues.password}
+                          onChange= { handleChange }
+                          error = { formErrors.password }
+                          helperText= { formErrors.password || null}
+                        />
+                      </Grid>
+                      <Grid item xs={12}>
+                        <TextField
+                          required
+                          fullWidth
+                          name="confirmPassword"
+                          label="Confirm Password"
+                          type="password"
+                          id="confirmPassword"
+                          autoComplete="current-password"
+                          InputProps = {{ 
+                            classes: {root: classes.textFieldSelectLabel} 
+                          }}
+                          InputLabelProps = {{ 
+                            classes: {root: classes.textFieldSelectLabel} 
+                          }}
+                          value={formValues.confirmPassword}
+                          onChange = { handleChange }
+                          error = { formErrors.confirmPassword }
+                          helperText= { formErrors.confirmPassword || null}
                         />
                       </Grid>
                       <Grid item xs={12}>
                         <FormControl className={classes.formControl}>
-                          <InputLabel shrink id="demo-simple-select-placeholder-label-label">
-                            Age
+                          <InputLabel classes = {{ root: classes.selectRoot }} shrink id="demo-simple-select-placeholder-label-label">
+                            Select Primary Language
                           </InputLabel>
                           <Select
+                            fullWidth
                             labelId="demo-simple-select-placeholder-label-label"
                             id="demo-simple-select-placeholder-label"
-                            value={age}
+                            value={formValues.language}
                             onChange={handleChange}
-                            displayEmpty
                             className={classes.selectEmpty}
+                            classes = {{ select: classes.selectTopPadding}}
                           >
-                            <MenuItem value="">
-                              <em>None</em>
-                            </MenuItem>
-                            <MenuItem value={10}>Ten</MenuItem>
-                            <MenuItem value={20}>Twenty</MenuItem>
-                            <MenuItem value={30}>Thirty</MenuItem>
+                            <MenuItem value={10}> English </MenuItem> 
+                            <MenuItem value={20}>Mandarin</MenuItem>
```

Comment:
will probably want to set the value to "english", "mandarin", etc

Reply:
Thanks for suggesting this approach. Works great!